src/library/dlgtagfetcher.h

```diff
@@ -107,5 +111,7 @@ class DlgTagFetcher : public QDialog, public Ui::DlgTagFetcher {
 
     QByteArray m_fetchedCoverArtByteArrays;
 
+    QMap<QString, QPixmap> m_coverCache;
```

Comment:
Did you consider to extend use the exiting cover cache here? 
I have not looked into it, but I can imagine that this way we can save one cover access. 

An other option is to use QPixmapCache or the general purpose QCache container.

Reply:
In the comments above I explained why QPixmapCache is unsuitable, and that I didn't get it to work with QCache (even though QPixmapCache is derived from QCache). Any idea how to fix the QCache implementation is welcome, even though the only benefits are slightly faster lookup and automatic cleanup (limit).